packages/block-editor/src/components/block-list-appender/style.scss

```diff
@@ -1,17 +1,50 @@
+.block-list-appender {
+	margin: $block-padding;
+
+	// Certain Blocks (eg: Column) act as as a "passthrough container"
+	// and therefore have their vertical margins/padding removed via negative margins
+	// therefore we need to compensate for this here by doubling the spacing on the
+	// vertical to ensure there is equal visual spacing around the inserter. Note there
+	// is no formal API for a passthrough Block but in lieu of that a `is-` class should
+	// suffice here
+	.is-passthrough & {
```

Comment:
Just wondering about this, and whether there's a better way.

I think it might be preferable if there was a way to declare this on the appender rather than as a class on the parent block.

Doing that isn't straightforward though. The only way I can think to achieve this is to pass extra data through the prop:
```
<InnerBlocks
    appender={ {
        type="button"
        offsetNegativeMargin={ true }
    } }
    templateLock={ false }
/>
```

Reply:
Another option could be to use a slot/fill for the appender ...

Reply:
> Doing that isn't straightforward though. The only way I can think to achieve this is to pass extra data through the prop:

I'm thinking we should try and avoid overcomplicating the API here if possible.

Perhaps edge cases such as Column should add their own styling to `block-list-appender` directly. This would avoid the `block-list-appender` having too much knowledge about its usage and surroundings and maintain SRP.

Reply:
I've reverted the `is-passthrough` route and gone for adding an edge case overide to Columns.

I think this is entirely valid because whilst Columsn won't have the placeholder enabled by default when we merge this PR, we are opening up that possibility. Therefore we ought to account for it.